apps/omg_child_chain_rpc/lib/omg_child_chain_rpc/web/controllers/transaction.ex

```diff
@@ -18,7 +18,8 @@ defmodule OMG.ChildChainRPC.Web.Controller.Transaction do
   """
 
   use OMG.ChildChainRPC.Web, :controller
-  plug(OMG.ChildChainRPC.Plugs.Health)
+  # check for health before calling action
+  # plug(OMG.ChildChainRPC.Plugs.Health)
```

Comment:
Was you intention to leave it commented?

Reply:
it used to be enabled, so I commented it out so that we could have a discussion on what to do, cc @pdobacz